container/Dockerfile

```diff
@@ -9,90 +8,48 @@ ARG TRAVIS_BUILD_NUMBER
 ARG TRAVIS_BUILD_WEB_URL
 ARG IMAGE_NAME
 
+ARG DOCKER_CLIENT_VERSION=23.0
+ARG DOCKER_COMPOSE_VERSION=2.17
+ARG KUBECTL_VERSION=1.27
+ARG HELM_VERSION=3.11
+
 LABEL git.branch=${GIT_BRANCH}
 LABEL git.commit.id=${GIT_COMMIT_ID}
 LABEL git.dirty=${GIT_DIRTY}
 LABEL git.build.time=${GIT_BUILD_TIME}
 LABEL ci.build.number=${TRAVIS_BUILD_NUMBER}
 LABEL ci.build.web.url=${TRAVIS_BUILD_WEB_URL}
 
-COPY --from=docker/compose:alpine-1.26.2 /usr/local/bin/docker-compose /usr/local/bin/
+COPY --from=docker/compose:alpine-1.29.2 /usr/local/bin/docker-compose /usr/local/bin/
```

Comment:
So we will have docker-compose and docker compose. Is it better to have an alias `docker-compose=docker compose`

Reply:
Yes, currently the job-engine actions are still using `docker-compose`.
We cannot alias it because some parameters are different and their position are different as well.

For example:
```
docker-compose -H 1.2.3.4 up -d
```
should be translated to:
```
docker -H 1.2.3.4 compose up -d
```
but with an alias it will be
```
docker compose -H 1.2.3.4 up -d
```
which will fail

There is a tool called [compose-switch](https://github.com/docker/compose-switch) which is supposed to do a proper translation but @ignacio-penas tried it and it didn't worked as expected.